school/src/main/java/com/hogwarts/school/controller/StudentController.java

```diff
@@ -19,13 +21,12 @@ public StudentController(StudentService studentService) {
 
     @PostMapping("/addStudent")
```

Comment:
```suggestion
    @PostMapping
```
не нужно ничего писать в пути - это ошибка REST